### PR TITLE
mu4e: Improve mu4e manual.

### PR DESCRIPTION
**Pull request by tychobrailleur**
Fix some typos, improve formatting, and move down `@documentencoding`
otherwise causing issues on the PDF cover.

### CODE REVIEW

File: mu4e/mu4e.texi [hunk 1]
```diff
@@ -85,7 +85,7 @@ you up to speed quickly: @ref{Example configurations}. There's also a
 section with answers to frequenly asked questions, @ref{FAQ}.
 
 @menu
-* Introduction:: Where be begin
+* Introduction:: Where we begin
```
Comment (Chris00) [inline mu4e/mu4e.texi]: Wouldn't it be better to write "_Where to begin_"?

File: mu4e/mu4e.texi [hunk 2]
```diff
@@ -25,7 +25,7 @@ Documentation License.''
 @end copying
 
 @titlepage
-@title @t{Mu4e} - an e-mail client for GNU/Emacs
+@title @t{Mu4e} -- an e-mail client for GNU/Emacs
```
Comment (Chris00) [inline mu4e/mu4e.texi]: Shouldn't it be `---` ("—" instead of "–")?

File: mu4e/mu4e.texi [hunk 3]
```diff
@@ -70,8 +70,8 @@ queries.
 @item Support for non-English languages (so ``angstrom''  matches ``Ångström'')
 @item Asynchronous: heavy actions don't block @t{emacs}@footnote{currently,
 the only exception to this is @emph{sending mail}; there are solutions
-for that though - see the @ref{FAQ}}
-@item Support for cryptography - signing, encrypting and decrypting
+for that though -- see the @ref{FAQ}}
+@item Support for cryptography -- signing, encrypting and decrypting
```
Comment (Chris00) [inline mu4e/mu4e.texi]: Use `---`, the [emdash](http://www.thepunctuationguide.com/em-dash.html) (the [endash](http://www.thepunctuationguide.com/en-dash.html) "–" serves to denote ranges)

File: mu4e/mu4e.texi [hunk 4]
```diff
@@ -115,7 +115,7 @@ Let's get started!
 
 @menu
 * Why another e-mail client::Aren't there enough already
-* Other mail clients::Where mu4e takes its inspiration
+* Other mail clients::Where @t{mu4e} takes its inspiration
```
Comment (Chris00) [inline mu4e/mu4e.texi]: "Where @t{mu4e} takes its inspiration _from_"?

File: mu4e/mu4e.texi [hunk 5]
```diff
@@ -248,7 +248,7 @@ you also need the development packages for GTK+, Webkit and Guile.
 @node Installation
 @section Installation
 
-@t{mu4e} is part of @t{mu} - by installing the latter, the former is installed
+@t{mu4e} is part of @t{mu} -- by installing the latter, the former is installed
```
Comment (Chris00) [inline mu4e/mu4e.texi]: `---`

File: mu4e/mu4e.texi [hunk 6]
```diff
@@ -332,7 +332,7 @@ $ sudo make install
 (Xapian, GMime and their dependencies must be installed).
 
 After this, @t{mu} and @t{mu4e} should be installed @footnote{there's a hard
-dependency between versions of @t{mu4e} and @t{mu} - you cannot combine
+dependency between versions of @t{mu4e} and @t{mu} -- you cannot combine
```
Comment (Chris00) [inline mu4e/mu4e.texi]: `---` (and below, I won't say it again).

File: mu4e/mu4e.texi [hunk 7]
```diff
@@ -381,7 +381,7 @@ should be able to provide you with the details.
 @section Indexing your messages
 
 After you have succeeded in @ref{Getting mail}, we need to @emph{index} the
-messages. That is - we need to scan the messages in the maildir and store the
+messages. That is -- we need to scan the messages in the maildir and store the
 information about them in a special database. We can do that from @t{mu4e} --
```
Comment (Chris00) [inline mu4e/mu4e.texi]: `--` → `---`

File: mu4e/mu4e.texi [hunk 8]
```diff
@@ -555,21 +555,21 @@ influence, @code{mu4e-get-mail-command} based on the the current
...
-updates - for example when receiving new mail. See
+updates -- for example when receiving new mail. See
 @code{mu4e-index-updated-hook} and some tips on its usage in the
 @ref{FAQ}.
 
 @node Sending mail
 @section Sending mail
 
-@t{mu4e} re-uses Gnu's @code{message-mode} (@inforef{Top,,message}) for
+@t{mu4e} re-uses Gnus's @code{message-mode} (@inforef{Top,,message}) for
```
Comment (Chris00) [inline mu4e/mu4e.texi]: `Gnus'`  (there is already an `s` so `'s` becomes `'` I think).

File: mu4e/mu4e.texi [hunk 9]
```diff
@@ -2272,30 +2272,30 @@ in the list @code{mu4e-marks}. Such an element must have the following form:
...
-(The message is passed as MSG to the function). This function allows
-to compute a per-message target, for refile-like marks. If nil, the
-DYN-TARGET passed to the @code{:action} is the TARGET obtained as above.
+(The message is passed as @t{MSG} to the function). This function allows
+to compute a per-message target, for refile-like marks. If @t{nil}, the
+@t{DYN-TARGET} passed to the @code{:action} is the @t{TARGET} obtained as above.
 @item @code{:show-target} -- how to display the target in the headers view.
-If @code{:show-target} is nil the DYN-TARGET is shown (and DYN-TARGET must be
-a string).
+If @code{:show-target} is @t{nil} the @t{DYN-TARGET} is shown (and
```
Comment (Chris00) [inline mu4e/mu4e.texi]: Shouldn't `@t{nil}` be `@code{nil}`?

File: mu4e/mu4e.texi [hunk 10]
```diff
@@ -4427,7 +4429,7 @@ details that are subject to change; and see the other convenience functions in
 Some notes on the format:
 @itemize
 @item The address fields are @emph{lists} of pairs @code{(name . email)},
-where @t{name} can be nil.
+where @t{name} can be @t{nil}.
```
Comment (Chris00) [inline mu4e/mu4e.texi]: `@code{nil}`?

File: mu4e/mu4e.texi (hunk 2 above)
Comment (tychobrailleur) [inline mu4e/mu4e.texi]: If we stick to US English, I believe you're correct, an em-dash—without surrounding whitespaces—would be more appropriate.  However, I used en-dashes for consistency with the rest of the document where en-dashes seem to be more prevalent.

If there's consensus for “—”, I will change everywhere.

File: mu4e/mu4e.texi (hunk 8 above)
Comment (tychobrailleur) [inline mu4e/mu4e.texi]: I'll change that.

File: mu4e/mu4e.texi (hunk 9 above)
Comment (tychobrailleur) [inline mu4e/mu4e.texi]: I followed the convention used in the rest of the doc here too. ;-)

**Comment by tychobrailleur** [comment]
Thanks for the review!  Let me know if we want to switch to em-dashes, I'll do the change, and squash the commits together.

**Comment by djcb** [comment]
This has been merged, thanks! Including @Chris00's `---` suggestion.

**Pull request closed without merging**